### DISCUSSION TITLE
openwrt 下解压安装包报错

### DISCUSSION THREAD

**Discussion by rockmetalcn** [XIU2/CloudflareSpeedTest#147] [提问]
wget https://github.com/XIU2/CloudflareSpeedTest/releases/download/v2.0.2/CloudflareST_linux_arm64.tar.gz
root@OpenWrt:/CloudflareST# tar -zxf CloudflareST_linux_arm64.tar.gz 
tar: invalid tar magic

在软路由上解压总报错 压缩格式问题 帮忙看一下啊 谢谢！

**Comment by yyuurrii34** [comment]
openwrt 里的tar不带gzip和xz组件的，所以解压tar.gz或tar.xz时要分两步解压：
gzip -d *.tar.gz
#xz -d *.tar.xz
tar -xvf *.tar
